scripts/scons_plugin/thirdparty.py

```diff
@@ -79,22 +79,22 @@ def needlib(lib):
         includes = ['']
 
     for s in includes:
-        incdir  ='%s/%s/include' % (build_root, versioned_name)
+        incdir = os.path.join(build_root, versioned_name, 'include')
         if s:
-            incdir += '/' + s
+            incdir = os.path.join(incdir, s)
 
         if os.path.isdir(env.Dir(incdir).abspath):
             env.Prepend(CPPPATH=[incdir])
 
-    libdir = '%s/%s/lib' % (build_root, versioned_name)
-    rpathdir = '%s/%s/rpath' % (build_root, versioned_name)
+    libdir = os.path.join(build_root, versioned_name, 'lib')
+    rpathdir = os.path.join(build_root, versioned_name, 'rpath')
```

Comment:
Here and in other places, let's revert os.path.join.

These paths are passed to scons and I think they should use `/`